automatic/git.portable/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,32 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$is64bit = Get-ProcessorBits 64
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$filePath32 = "$toolsDir\PortableGit-2.11.0-32-bit.7z.exe"
+$filePath64 = "$toolsDir\PortableGit-2.11.0-64-bit.7z.exe"
+$filePath = if ($is64bit -and $env:chocolateyForceX86 -eq $false) { $filePath64 } else { $filePath32 }
```

Comment:
Notice that I use wildcars above, but its probably better to use full name. (if removing older installer fails, gi will return multiple files).

 I use wildcards so that same installer works with other tools.